apiserver/facades/client/modelmanager/modelinfo_test.go

```diff
@@ -344,7 +346,7 @@ func (s *modelInfoSuite) expectedModelInfo(c *gc.C, credentialValidity *bool) pa
 func (s *modelInfoSuite) TestModelInfo(c *gc.C) {
 	api, ctrl := s.getAPI(c)
 	defer ctrl.Finish()
-	s.mockAccessService.EXPECT().GetModelUsers(gomock.Any(), user.AdminUserName, coremodel.UUID(s.st.model.cfg.UUID())).Return(s.modelUserInfo, nil)
+	s.mockAccessService.EXPECT().GetModelUsers(gomock.Any(), gomock.Any(), coremodel.UUID(s.st.model.cfg.UUID())).Return(s.modelUserInfo, nil)
```

Comment:
What's the rationale for this change? Better to assert the username going in is correct if possible IMO?

Reply:
There are a lot of other similar changes in this file

Reply:
Ah, yea, so this is beacuse I made the changes for the other PR then separated them out here to make it easier to review. In the end though, I'm not sure it actually makes things better, I think I might just close this PR and say that this is just a commit in #18003.

A lot of the changes in here will be changed again with the changes to GetModelUsers in there.